aiohttp/web_urldispatcher.py

```diff
@@ -145,7 +145,7 @@ class StaticRoute(Route):
 
     def __init__(self, name, prefix, directory, *,
                  expect_handler=None, chunk_size=256*1024,
-                 response_factory=None):
+                 response_factory=None, sendfile_fallback=False):
```

Comment:
Fallback should be performed automatically.
User doesn't know (and doesn't care) is underlying OS support sendfile.